docs/api/python/dev.rst

```diff
@@ -28,9 +28,9 @@ tvm.expr
    :members:
    :undoc-members:
 
-tvm.codegen
-~~~~~~~~~~~
-.. automodule:: tvm.codegen
+tvm.target.codegen
+~~~~~~~~~~~~~~~~~~
```

Comment:
can ignore changes for now, superceded by https://github.com/apache/incubator-tvm/pull/4907

Reply:
fixed